### ISSUE TITLE
Device brightness goes to minimum when video is rotated to landscape mode

### ISSUE THREAD

**Issue by captainnapalm** [libre-tube/LibreTube#3362]
### Steps to reproduce

1. Start playing a video
2. Rotate phone to landscape

### Expected behavior

Video continues playing at existing device brightness

### Actual behavior

Video rotates properly but immediately the device brightness goes to minimum. Rotating the device back to portrait brings the device brightness back to normal. This only happens on my device in LibreTube. Other video apps like Jellyfin, Plex, Youtube, etc do not exhibit this behavior when the device is rotated to landscape.

### LibreTube version

0.12.1

### Android version

Android 12

### Other details

This is on a Fairphone 4 5G with app installed via F-droid (Droid-ify).

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by akano12** [comment]
LT plays the video with last remembered brightness level.
Try:
Play a video -> Change brightness to whatever level -> Close the video -> Play another video and check if issue persists

**Comment by captainnapalm** [comment]
Still persists after changing brightness during playback, and starting a new video, or force closing the app and starting a new video.

The actual device brightness isn't modified, as you suggest. It was at the correct level for normal usage when I pulled the notification shade down, and modifying it with the shade open did work, but once the shade was closed again and I returned to the video, the brightness dimmed again.
